eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -306,10 +306,13 @@ function DeleteOrUpdateResourceGroups() {
       }
       continue
     }
-    # TODO: Remove $true and follow non-compliant group deletion
+    if (!$DeleteNonCompliantGroups) {
+      continue
+    }
+    # TODO: Follow non-compliant group deletion for ALL managed subscriptions
     # Currently this is disabled in order to roll out features of the script slowly.
     # See https://gitub.com/Azure/azure-sdk-tools/issues/2714h
-    if ($true -or !$DeleteNonCompliantGroups) {
+    if ($SubscriptionId -ne "2cd617ea-1866-46b1-90e3-fffb087ebf9b") {  # Azure SDK Test Resources
```

Comment:
Is the goal to test and then remove this check?

Reply:
Playground has a longer tail of things that would be deleted, so I want to give a more formal heads up for them. The imminent issue is all the azure dev and mgmt test groups that are clogging up our main test sub.

Reply:
Why would you be using the playground subid here then to ski?

Reply:
The way it behaves here is we will only add the `DeleteAfter` tag when missing for the Test Resources sub. I haven't dug into the other subscriptions yet to see if they have any static groups that are required (e.g. ACS test subs, storage test subs, sov cloud). So I'd rather do it as an opt-in vs. opt-out.

Reply:
Oh this loop is just adding to the update list. So right now that means we are only doing it for our one test subscription. At some point we will want to do that for every subscription we clean-up. Should we instead control that with a flag we pass to the script for each sub-config?

Reply:
e.g. an array of sub IDs to authorize?